ImgSmall.java

```diff
@@ -0,0 +1,65 @@
+import javax.imageio.ImageIO;
+import java.awt.Image;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.nio.file.Paths;
+
+public class ImgSmall {
+  public static void main(String[] args) throws IOException {
```

Comment:
I think I mentioned before. Do not throw IOException. 
If an IOException occurs - likely at ImageIO.read() call then catch it and print a sensible error like - sorry unable to read the image file, please try again. 
Also when you catch the Exception you can call methods on Exception to get some useful information like reason 
catch (IOException ioe) {
  System.err(ioe.toString())  
  // read https://docs.oracle.com/javase/7/docs/api/java/io/IOException.html 
  // and methods inherited from Throwable
}